pages/common/eslint.md

```diff
@@ -0,0 +1,19 @@
+# eslint
+
+> Find problematic patterns or code that doesn’t adhere to certain style guidelines.
+
+- Create eslint config:
+
+`eslint --init`
+
+- Fix lint:
+
+`eslint --fix`
+
+- Lint on files:
```

Comment:
Put this example before `--fix`. And since you are saying "files", please add another file too. So that its clear that the tool can work on multiple files.